docs/src/intro.md

```diff
@@ -59,7 +59,7 @@ Developers can create CLI profiles to install multiple individual CLI's at the s
 
 ## Documentation
 
-Visit the [Charmil Documentation](docs/src)
+Visit the [Charmil Documentation](https://github.com/aerogear/charmil/blob/main/starter/CONTRIBUTING.md#documentation)
```

Comment:
```suggestion
Visit the [Charmil Documentation](https://aerogear.github.io/charmil/docs/)
```
The path you've added points to the CONTRIBUTING.md file, instead we need to point it to the website docs (as mentioned in issue #229)

Reply:
oh sorry , i've committed your suggestion so now hopefully it is guiding to the right place